tests/test_integration_ftp.py

```
@@ -0,0 +1,485 @@
+                instrument=instrument,
+                charge=charge,
+                **strategy_params
+            )
+            
+            msp_files = list(output_dir.glob("*.msp"))
+            if len(msp_files) > 0:
+                logger.info(f"Strategy '{strategy_name}' created {len(msp_files)} MSP file(s)")
+            else:
+                logger.warning(f"Strategy '{strategy_name}' did not create any MSP files")
+                
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)